plugin site is failing to build on ci

```
failed Building static HTML for pages - 1.627s
error Building static HTML failed for path "/aws-secrets-manager-credentials-provider/"

   7 |     return (
   8 |         <p className="box">
>  9 |             <a href={reportProblemUrl.includes('://') ? reportProblemUrl : pluginSiteReportUrl} title={title}>
     |                                       ^
  10 |                 Report a problem
  11 |             </a>
  12 |         </p>

[0m
[0m  [0m[97m[41mWebpackError[0m[37m[41m:[0m[37m[41m [0m[97m[41mTypeError: Cannot read property 'includes' of null[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mReportAProblem.jsx[0m[90m:[0m[93m9[0m[37m [0m
[0m  [0m  [0m[90mjenkins-plugin-site/src/components/ReportAProblem.jsx:9:39[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mcache-common.esm.js[0m[90m:[0m[93m5[0m[37m [0m
[0m  [0m  [0m[90m[jenkins-plugin-site]/[@algolia]/cache-common/dist/cache-common.esm.js:5:1[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mcache-common.esm.js[0m[90m:[0m[93m4[0m[37m [0m
[0m  [0m  [0m[90m[jenkins-plugin-site]/[@algolia]/cache-common/dist/cache-common.esm.js:4:36[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mcache-common.esm.js[0m[90m:[0m[93m15[0m[37m [0m
[0m  [0m  [0m[90m[jenkins-plugin-site]/[@algolia]/cache-common/dist/cache-common.esm.js:15:10[0m  [0m
[0m  [0m[90m-[0m [0m[93mstatic-entry.js[0m[90m:[0m[93m286[0m[37m [0m
[0m  [0m  [0m[90mjenkins-plugin-site/.cache/static-entry.js:286:22[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mclient-common.esm.js[0m[90m:[0m[93m74[0m[37m [0m
[0m  [0m  [0m[90m[jenkins-plugin-site]/[@algolia]/client-common/dist/client-common.esm.js:74:[0m  [0m  [0m[90m1[0m
[0m  [0m
[0m
not finished Caching JavaScript and CSS webpack compilation - 12.758s
not finished Caching HTML renderer compilation - 1.764s
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! jenkins-plugin-site@0.0.1 build: `gatsby build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the jenkins-plugin-site@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jenkins/agent/workspace/plugin-site_master/.npm/_logs/2021-09-11T09_05_14_299Z-debug.log
```

Full stack trace
```
error Page data from page-data.json for the failed page "/aws-secrets-manager-credentials-provider/": {
  "componentChunkName": "component---src-templates-plugin-jsx",
  "path": "/aws-secrets-manager-credentials-provider/",
  "result": {
    "data": {
      "jenkinsPlugin": {
        "id": "aws-secrets-manager-credentials-provider",
        "gav": "io.jenkins.plugins:aws-secrets-manager-credentials-provider:0.5.3",
        "hasBomEntry": false,
        "title": "AWS Secrets Manager Credentials Provider",
        "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.5.3/aws-secrets-manager-credentials-provider.hpi",
        "version": "0.5.3",
        "wiki": {
          "content": "<div id=\"readme\" class=\"md\" data-path=\"docs/README.md\">\n <article class=\"markdown-body entry-content container-lg\" itemprop=\"text\"> \n  <p><a href=\"https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Faws-secrets-manager-credentials-provider-...",
          "url": "https://github.com/jenkinsci/aws-secrets-manager-credentials-provider-plugin"
        },
        "stats": {
          "currentInstalls": 1936,
          "installations": [
            {
              "timestamp": 1598918400000,
              "total": 919
            },
            {
              "timestamp": 1601510400000,
              "total": 1069
            },
            {
              "timestamp": 1604188800000,
              "total": 1089
            },
            {
              "timestamp": 1606780800000,
              "total": 1196
            },
            {
              "timestamp": 1609459200000,
              "total": 1322
            },
            {
              "timestamp": 1612137600000,
              "total": 1432
            },
            {
              "timestamp": 1614556800000,
              "total": 1700
            },
            {
              "timestamp": 1617235200000,
              "total": 1623
            },
            {
              "timestamp": 1619827200000,
              "total": 1634
            },
            {
              "timestamp": 1622505600000,
              "total": 1784
            },
            {
              "timestamp": 1625097600000,
              "total": 1798
            },
            {
              "timestamp": 1627776000000,
              "total": 1936
            }
          ],
          "trend": 218
        },
        "sha1": "TwSV2zVsAzgNEcN88zFbFwUNgGc=",
        "deprecationNotice": null,
        "securityWarnings": [],
        "issueTrackers": [
          {
            "reportUrl": null,
            "viewUrl": "https://issues.jenkins.io/issues/?jql=component=25223"
          },
          {
            "reportUrl": "https://github.com/jenkinsci/aws-secrets-manager-credentials-provider-plugin/issues/new/choose",
            "viewUrl": "https://github.com/jenkinsci/aws-secrets-manager-credentials-provider-plugin/issues"
          }
        ],
        "scm": "https://github.com/jenkinsci/aws-secrets-manager-credentials-provider-plugin/tree/master/blob/master/docs/README.md",
        "hasPipelineSteps": false,
        "requiredCore": "2.222.4",
        "releaseTimestamp": "2021-01-21T14:48:03.00Z",
        "previousVersion": "0.5.2",
        "previousTimestamp": "2021-01-07T13:43:59.00Z",
        "name": "aws-secrets-manager-credentials-provider",
        "labels": [
          "aws"
        ],
        "developers": [
          {
            "id": "chriskilding",
            "name": "Chris Kilding"
          }
        ],
        "firstRelease": "2019-06-26T00:00:00.000Z",
        "excerpt": "Source Jenkins Credentials from AWS Secrets Manager.",
        "buildDate": "Jan 21, 2021",
        "dependencies": [
          {
            "implied": null,
            "name": "configuration-as-code",
            "optional": true,
            "title": "Configuration as Code",
            "version": "1.44"
          },
          {
            "implied": null,
            "name": "aws-java-sdk",
            "optional": false,
            "title": "Amazon Web Services SDK",
            "version": "1.11.930"
          },
          {
            "implied": null,
            "name": "credentials",
            "optional": false,
            "title": "Credentials",
            "version": "2.3.13"
          },
          {
            "implied": null,
            "name": "plain-credentials",
            "optional": false,
            "title": "Plain Credentials",
            "version": "1.7"
          },
          {
            "implied": null,
            "name": "ssh-credentials",
            "optional": false,
            "title": "SSH Credentials",
            "version": "1.18.1"
          },
          {
            "implied": true,
            "name": "sshd",
            "optional": false,
            "title": "SSH server",
            "version": "3.0.1"
          }
        ]
      },
      "versions": {
        "edges": [
          {
            "node": {
              "buildDate": "2021-01-21T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.44"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.930"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.13"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.7"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.5.3",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.222.4",
              "sha1": "TwSV2zVsAzgNEcN88zFbFwUNgGc=",
              "sha256": "Jd2e3rcjGVuUA8EgCZayntfvLe/sdUZSN7EGxkAWVaw=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.5.3/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.5.3"
            }
          },
          {
            "node": {
              "buildDate": "2021-01-07T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.44"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.930"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.13"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.7"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.5.2",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.222.4",
              "sha1": "b5RSIZY8EdDkSyl2zk+S9GZrX9o=",
              "sha256": "iWluCSL93KT1DaaLngAnwy5R3ltdqAw1K79YdC0lG3Q=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.5.2/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.5.2"
            }
          },
          {
            "node": {
              "buildDate": "2020-11-09T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.43"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.821"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.13"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.7"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.5.1",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.222.4",
              "sha1": "XMY333M1HUyUZ3U1my1Ui5uqM20=",
              "sha256": "vsipabjDX44+s5mp/DeqekOY4jOQf/7ELfYeDGvvRWY=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.5.1/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.5.1"
            }
          },
          {
            "node": {
              "buildDate": "2020-10-22T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.36"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.821"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.11"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.7"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.5.0",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.204.6",
              "sha1": "bw04K2rmh/ICHyPhhxoYBBpDUDw=",
              "sha256": "934ZoUtyp54sU+0MqEnuaW99ZFSUGGTdFjds7EhhJAc=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.5.0/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.5.0"
            }
          },
          {
            "node": {
              "buildDate": "2020-10-16T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.36"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.821"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.11"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.7"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.4.2",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.204.6",
              "sha1": "1SzoPEsMDdtgLO8iwJXvxAben/s=",
              "sha256": "BpI9kRs/IoJ76pU3kuu0xTjfFFim283n0IfM+o2uw4I=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.4.2/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.4.2"
            }
          },
          {
            "node": {
              "buildDate": "2020-09-17T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.36"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.821"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.9"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.6"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.4.1",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.204.6",
              "sha1": "Qs5zDhIQEBx5gzZAQDBbB0+xaA4=",
              "sha256": "PyrKVfUgD3L3oOvWrwDCFnm4tOGQ5/VY7LO5t67Imb0=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.4.1/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.4.1"
            }
          },
          {
            "node": {
              "buildDate": "2020-09-10T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.36"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.821"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.9"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.6"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.4.0",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.204.6",
              "sha1": "WWhVCgHykIuax7sKt4QgJ6UabNs=",
              "sha256": "7l6r9+HdvY/UN9YJ5uXrFADBchCMQo77dlkMAvHcAdU=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.4.0/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.4.0"
            }
          },
          {
            "node": {
              "buildDate": "2020-08-18T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.36"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.821"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.9"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.6"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.3.3",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.204.6",
              "sha1": "ceUJb9Zi+rvKVWaohquUTrb5CaQ=",
              "sha256": "pCaUu8aZ6LjzKTadjOMnGwIpRF5ez1xlSrux+zBZzJI=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.3.3/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.3.3"
            }
          },
          {
            "node": {
              "buildDate": "2020-07-23T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.36"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.821"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.11"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.7"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.3.2",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.204",
              "sha1": "dwBRm3wg8kBk4p/PLbj7JcHi3SI=",
              "sha256": "6P0dNCcNN3LqJ+n5Y6HubYPFLdgKNX2nLEROBmWi4tk=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.3.2/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.3.2"
            }
          },
          {
            "node": {
              "buildDate": "2020-07-20T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.36"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.821"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.11"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.7"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.18.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.3.1",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.204",
              "sha1": "tqrKNXOVQ+PJCi9cYG4tvCQZShE=",
              "sha256": "dbfOlBpGLdBT0mLeVPTHtl0Vhw4hjv0i0OTV2UF5SpQ=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.3.1/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.3.1"
            }
          },
          {
            "node": {
              "buildDate": "2020-06-22T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": true,
                  "name": "configuration-as-code",
                  "version": "1.35"
                },
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.636"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.3.0",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "2I5S6KWHiA/LcvbNAPNjLteIppM=",
              "sha256": "QXNTIO1KzmeMDLN1x/hv3+rvmH/VDZ8qrPA2rnN1SkI=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.3.0/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.3.0"
            }
          },
          {
            "node": {
              "buildDate": "2020-02-21T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.636"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.2.2",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "v6V2an4SCBmu+d/DtC7iK9WNiPo=",
              "sha256": "TwzmeyzOb7613YBLLk0pRZXKfX2kd3fh67vuyfqf3zA=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.2.2/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.2.2"
            }
          },
          {
            "node": {
              "buildDate": "2020-01-27T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.636"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.2.1",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "Fk2ivucogIFSxRHj0BpXx6P3Vro=",
              "sha256": "C8ckBurf1AALajUlrnzNfyu8tgRrui1O1V7qos0RsQ8=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.2.1/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.2.1"
            }
          },
          {
            "node": {
              "buildDate": "2020-01-23T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.636"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.1"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.2.0",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "kvRT6wQZbfLWrTn9RSzH8qpFF78=",
              "sha256": "tEcy/NAcUjL9m1bH/qb9E9PpHFam9iye0O3xZScSzh0=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.2.0/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.2.0"
            }
          },
          {
            "node": {
              "buildDate": "2020-01-21T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.636"
                },
                {
                  "optional": false,
                  "name": "bouncycastle-api",
                  "version": "2.17"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.1"
                },
                {
                  "optional": true,
                  "name": "pipeline-model-definition",
                  "version": "1.3.9"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.1.4",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "lAH2Xx1FvFQb/Jz6zb+BsjDMEM4=",
              "sha256": "/5PKf3vSDBpxA2KxpsBaoF5rBzk1/9nOLrEjJQMOEus=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.1.4/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.1.4"
            }
          },
          {
            "node": {
              "buildDate": "2020-01-20T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.636"
                },
                {
                  "optional": false,
                  "name": "bouncycastle-api",
                  "version": "2.17"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.1"
                },
                {
                  "optional": true,
                  "name": "pipeline-model-definition",
                  "version": "1.3.9"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.1.3",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "zOuX6OxRNSqIqoV8LonbJAx7oMw=",
              "sha256": "ID1Q5gRTZKJ/+bbY/D9Ye1JcZbgIAgz1HPfACQQZoqE=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.1.3/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.1.3"
            }
          },
          {
            "node": {
              "buildDate": "2019-12-16T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.636"
                },
                {
                  "optional": false,
                  "name": "bouncycastle-api",
                  "version": "2.17"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.1"
                },
                {
                  "optional": true,
                  "name": "pipeline-model-definition",
                  "version": "1.3.9"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.1.2",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "ocUd95/q5hty8GDCJMTayZUiO8o=",
              "sha256": "vJaHjh/Mvkyr/hi+ZWVq7Yzb4Br4nxee1znCxItf9y8=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.1.2/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.1.2"
            }
          },
          {
            "node": {
              "buildDate": "2019-12-06T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.562"
                },
                {
                  "optional": false,
                  "name": "bouncycastle-api",
                  "version": "2.17"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.1"
                },
                {
                  "optional": true,
                  "name": "pipeline-model-definition",
                  "version": "1.3.9"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.1.1",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "vXjQMLVcGRDpTTugzgD3DrwBtb4=",
              "sha256": "rbpup5abSfqpzBpuWpK7XzjPoJoIMaBIX3+pZ6Bnz0g=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.1.1/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.1.1"
            }
          },
          {
            "node": {
              "buildDate": "2019-11-28T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.636"
                },
                {
                  "optional": false,
                  "name": "bouncycastle-api",
                  "version": "2.17"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.3.0"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.5"
                },
                {
                  "optional": false,
                  "name": "ssh-credentials",
                  "version": "1.17.2"
                },
                {
                  "optional": true,
                  "name": "pipeline-model-definition",
                  "version": "1.3.9"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.1.0",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.176.4",
              "sha1": "A8BlJc83uXUiVW6OORMwdlRuKSs=",
              "sha256": "0ktBAc3QApV2SfzCcnpTT/13KmxU+VZK3lt1zOdrP24=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.1.0/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.1.0"
            }
          },
          {
            "node": {
              "buildDate": "2019-06-26T00:00:00.000Z",
              "compatibleSinceVersion": null,
              "dependencies": [
                {
                  "optional": false,
                  "name": "aws-java-sdk",
                  "version": "1.11.457"
                },
                {
                  "optional": false,
                  "name": "credentials",
                  "version": "2.1.16"
                },
                {
                  "optional": false,
                  "name": "plain-credentials",
                  "version": "1.4"
                }
              ],
              "id": "aws-secrets-manager-credentials-provider_0.0.1",
              "name": "aws-secrets-manager-credentials-provider",
              "requiredCore": "2.60.3",
              "sha1": "QMs7JgmC9nmuBVSgXKe9r24EW68=",
              "sha256": "H1omVVHUwEajr8+CDJCPOl+1zAK+bo8gntmrRqNyLD0=",
              "url": "https://updates.jenkins.io/download/plugins/aws-secrets-manager-credentials-provider/0.0.1/aws-secrets-manager-credentials-provider.hpi",
              "version": "0.0.1"
            }
          }
        ]
      },
      "reverseDependencies": {
        "edges": []
      }
    },
    "pageContext": {
      "name": "aws-secrets-manager-credentials-provider"
    }
  },
  "staticQueryHashes": [
    "4202924991",
    "443252636",
    "475252407"
  ]
}
failed Building static HTML for pages - 1.627s
error Building static HTML failed for path "/aws-secrets-manager-credentials-provider/"

   7 |     return (
   8 |         <p className="box">
>  9 |             <a href={reportProblemUrl.includes('://') ? reportProblemUrl : pluginSiteReportUrl} title={title}>
     |                                       ^
  10 |                 Report a problem
  11 |             </a>
  12 |         </p>

[0m
[0m  [0m[97m[41mWebpackError[0m[37m[41m:[0m[37m[41m [0m[97m[41mTypeError: Cannot read property 'includes' of null[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mReportAProblem.jsx[0m[90m:[0m[93m9[0m[37m [0m
[0m  [0m  [0m[90mjenkins-plugin-site/src/components/ReportAProblem.jsx:9:39[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mcache-common.esm.js[0m[90m:[0m[93m5[0m[37m [0m
[0m  [0m  [0m[90m[jenkins-plugin-site]/[@algolia]/cache-common/dist/cache-common.esm.js:5:1[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mcache-common.esm.js[0m[90m:[0m[93m4[0m[37m [0m
[0m  [0m  [0m[90m[jenkins-plugin-site]/[@algolia]/cache-common/dist/cache-common.esm.js:4:36[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mcache-common.esm.js[0m[90m:[0m[93m15[0m[37m [0m
[0m  [0m  [0m[90m[jenkins-plugin-site]/[@algolia]/cache-common/dist/cache-common.esm.js:15:10[0m  [0m
[0m  [0m[90m-[0m [0m[93mstatic-entry.js[0m[90m:[0m[93m286[0m[37m [0m
[0m  [0m  [0m[90mjenkins-plugin-site/.cache/static-entry.js:286:22[0m
[0m  [0m
[0m  [0m[90m-[0m [0m[93mclient-common.esm.js[0m[90m:[0m[93m74[0m[37m [0m
[0m  [0m  [0m[90m[jenkins-plugin-site]/[@algolia]/client-common/dist/client-common.esm.js:74:[0m  [0m  [0m[90m1[0m
[0m  [0m
[0m
not finished Caching JavaScript and CSS webpack compilation - 12.758s
not finished Caching HTML renderer compilation - 1.764s
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! jenkins-plugin-site@0.0.1 build: `gatsby build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the jenkins-plugin-site@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/jenkins/agent/workspace/plugin-site_master/.npm/_logs/2021-09-11T09_05_14_299Z-debug.log
```

--- a/src/templates/plugin.jsx
+++ b/src/templates/plugin.jsx
@@ -47,7 +47,7 @@ function PluginPage({data: {jenkinsPlugin: plugin, reverseDependencies: reverseD
 
     return (
         <Layout id="pluginPage" reportProblemRelativeSourcePath={pluginPage} reportProblemTitle={plugin.title}
-            reportProblemUrl={plugin.issueTrackers && plugin.issueTrackers[0] ? plugin.issueTrackers[0].reportUrl : `/${plugin.name}`}>
+            reportProblemUrl={plugin?.issueTrackers?.find(tracker => tracker.reportUrl)?.reportUrl || `/${plugin.name}`}>
             <SEO title={cleanTitle(plugin.title)} description={plugin.excerpt} pathname={`/${plugin.id}`}/>
             <div className="title-wrapper">
                 <h1 className="title">

